nixos/modules/services/networking/mosquitto.nix

```diff
@@ -257,18 +257,28 @@ let
 
       users = mkOption {
         type = attrsOf userOptions;
-        example = { john = { password = "123456"; acl = [ "topic readwrite john/#" ]; }; };
+        example = { john = { password = "123456"; acl = [ "readwrite john/#" ]; }; };
         description = ''
           A set of users and their passwords and ACLs.
         '';
         default = {};
       };
 
+      omitPasswordAuth = mkOption {
```

Comment:
The old module had the option `allowAnonymous`.

Is there a reason to add name the new option differently? If it were the same it would require less changes from the user.

Reply:
`allowAnonymous` in the old config lost its special treatment and is now `settings.allow_anonymous`. this one does something different, it's roughly equivalent to `! checkPasswords` in the old module and has a new name because the default is now different (and arguably more in line with what people expect)